exceptions.txt

```diff
@@ -419,3 +419,6 @@
 294973: 'Home Fires UK',
 295438: 'Shark UK',
 148581: 'Strike Back Legacy',
+281714: 'Ballers', 'Ballers (2015)', 'Ballers (2014)',
```

Comment:
Should be:
281714: 'Ballers (2015)', 'Ballers (2014)'

'Ballers' is already the name of the show on theTVDB, so adding an exception to that makes SR search for the same  name 2x.